src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -71,25 +73,23 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // If initializing the first NN or reconciling the first NN or bootstrapping the first NN
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
```

Comment:
the NOPMD comments actually have an effect.  They indicate that PMD should be ignored on this line.  We have several lines which have "unless parens".   I can pull the comments and add the ignores in the xml config file.   Better is to not use unless parens.